src/components/header.js

```diff
@@ -1,65 +1,71 @@
 // External Dependencies
+import PropTypes from 'prop-types';
 import React from 'react';
 import { Link } from 'gatsby';
+import { withStyles } from '@material-ui/core/styles';
 
 // Internal Dependencies
 import ReactLogoSvg from '../images/react-logo.svg';
 
 // Local Variables
-const reactLogoStyles = {
-  width: 24,
-  height: 24,
-  transform: 'translateY(6px)',
-  marginRight: 8,
+const propTypes = {
+  classes: PropTypes.shape({
```

Comment:
unrelated, but I gotta say that one of my favorite things about typescript with react is not having to build these manually anymore 😄